Lib9c/TableData/Item/EquipmentItemSetEffectSheet.cs

```diff
@@ -88,7 +89,8 @@ public static List<EquipmentItemSetEffectSheet.Row> GetSetEffectRows(this Equipm
                     rows.Add(row);
```

Comment:
Is it okay we don't mind the order of `rows`?

Reply:
yes.